frontend/app/assets/javascripts/spree/frontend/checkout/address.js

```diff
@@ -42,7 +42,6 @@ Spree.ready(function ($) {
           zipcodeLabel.text(zipText)
           zipcodeInput.addClass('required')
         } else {
-          zipcodeInput.val('')
```

Comment:
@MatthewKennedy is this required?

Reply:
This just causes a small issue where by if someone has entered a zip code although it’s not required for their country, if they the. go back to edit the address on page load it blanks out the zip code.

If you were to have the zip code field removed from the DOM then that would be needed.

Reply:
This looks like its been merged from the other PR